src/pages/BrowseHapps/BrowseHapps.js

```diff
@@ -34,15 +37,15 @@ export default function BrowseHapps ({ history: { push } }) {
           key={happ.id} />)}
     </div>}
 
-    <Button onClick={goToPricing} styleName='pricing-button'>Manage Pricing</Button>
+    <Button primary wide onClick={goToPricing} styleName='pricing-button'>Manage Pricing</Button>
```

Comment:
The code as written is how I'd prefer we do it, but with Button as it currently is, shouldn't this be

`<Button variant='primary' wide>`

Reply:
Ha, nice catch! Thanks!